desktop/core/ext-py3/gunicorn-20.1.0/gunicorn/workers/geventlet.py

```diff
@@ -130,7 +130,7 @@ def is_already_handled(self, respiter):
         return super().is_already_handled(respiter)
 
     def init_process(self):
-        self.patch()
+        #self.patch()
```

Comment:
this calls 
--> line 124 above eventlet.monkey_patch(), then calls
--> https://github.com/cloudera/hue/blob/master/desktop/core/ext-py3/eventlet-0.30.2/eventlet/__init__.py#L55, then calls 
--> https://github.com/cloudera/hue/blob/master/desktop/core/ext-py3/eventlet-0.30.2/eventlet/patcher.py#L225-L364

So, I don't feel safe to remove monkey patch in gunicorn.

Reply:
To upgrade eventlet and work with Gunicorn, a fix need to patch gunicorn. Not sure if this is the one you have tried
Issue: https://github.com/benoitc/gunicorn/pull/2581
Fix: https://github.com/benoitc/gunicorn/commit/1299ea9e967a61ae2edebe191082fd169b864c64

Reply:
The fix https://github.com/eventlet/eventlet/commit/7aa50ad5 in eventlet was the reason why I pulled in the whole of eventlet library. If the new gunicorn is pulling in eventlet v0.32.0+, then we can do away with our copy of eventlet.